lib/serverless-tools/deployer/ecr_pusher.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+require "aws-sdk-ecr"
```

Comment:
Minor - this isn't used in this file, is it?

Reply:
Gah, totally, it gets injected in the `FunctionDeployer` - fixed!